content/02.introduction.md

```diff
@@ -119,9 +120,68 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+Monocytes, which include both dendritic cells and macrophages, are types of phagocytic white blood cells that are also professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface via Major Histocompatability Complex II (MHC-II) proteins.
+This allows the cell to break the pathogen into its component peptides, which allow the body to organize a response to the specific pathogen.
+
+The adaptive immune response is the second line of defense once a pathogen has entered the body.
+It, in turn, can be broadly classified into the humoral and cell-mediated responses. 
+The humoral response includes soluble proteins and antibodies that can neutralize and eliminate pathogens in circulation that have not yet entered cells, and it can also mediate the killing of pathogen-infected cells. 
+The cell-mediated response includes the killing of pathogen-infected cells by cytotoxic CD8+ T cells and the secretion of cytokines to drive inflammation and support immune cell activities.
+The immune response therefore describes the complex interplay between pathogens infecting hosts and hosts responding with a multifaceted immune defense; a pathogen's virulence depends on its success in evading the immune system or otherwise manipulating the host.
+Innovation in therapeutics often seeks to supplement or bolster the immune response in order to enhance the body's ability to eliminate a pathogen.
+However, disease symptoms can be caused either by a pathogen or by the immune response to the pathogen, and a host immune system's efforts to fight an infection can also sometimes cause significant damage to the host [@doi:978-0815332183; @doi:9780071283663].
+Therefore, understanding how the host responds to SARS-CoV-2 and when this response becomes dangerous is critical to the treatment of COVID-19 and to the identification and/or development of possible therapeutics.
+
+##### hCOV and the Immune Response
+
+Research in other coronaviruses that affect humans provides some indication of how SARS-CoV-2 infection proceeds in spite of the human immune response.
+While the first line of defense for the immune system is physical barriers such as the skin and mucus, viruses such as SARS-CoV that infect the epithelium are known to be adept at bypassing these defenses [@doi:10.1038/mi.2015.127]. 
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of detection by monoctyes.
+Double-stranded RNA (dsRNA), which is produced during the viral replication process, is a PAMP that would typically initiate a response from the innate immune system [@doi:10.3389/fimmu.2018.00829].
```

Comment:
dsRNA is already defined above and does not need to be redefined here and in line 166.